### ISSUE TITLE
[BUG] Error when launching the example dataset

### ISSUE THREAD

**Issue by mdmanurung** [czbiohub/excellxgene#1]
**Describe the bug**
I encountered the following error when running `excellxgene launch example-dataset` as shown in the README.
```
[cellxgene] Starting the CLI...
[cellxgene] Loading data from example-dataset.
Loading and precomputing layers necessary for fast differential expression and reembedding...
Layer X ...
ValueError: No threading layer could be loaded.
HINT:
Intel TBB is required, try:
$ conda/pip install tbb

During handling of the above exception, another exception occurred:

numba.core.errors.LoweringError: Failed in nopython mode pipeline (step: nopython mode backend)
No threading layer could be loaded.
HINT:
Intel TBB is required, try:
$ conda/pip install tbb

File "..\..\Miniconda3\envs\cxg\lib\site-packages\backend\server\data_anndata\anndata_adaptor.py", line 976:
def _fmt_swapper(indices,indptr,data,n):#x,y,d,ptr):
    <source elided>
    pair = np.zeros_like(indices)
    for i in prange(indptr.size):
    ^

During: lowering "id=2[LoopNest(index_variable = parfor_index.33, range = (0, $16load_attr.6, 1))]{24: <ir.Block at C:\Users\mdmanurung\Miniconda3\envs\cxg\lib\site-packages\backend\server\data_anndata\anndata_adaptor.py (976)>}Var(parfor_index.33, anndata_adaptor.py:976)" at C:\Users\mdmanurung\Miniconda3\envs\cxg\lib\site-packages\backend\server\data_anndata\anndata_adaptor.py (976)
```

**To Reproduce**
I followed the installation guide line by line, including the part where I need to install the tbb. However, I still encountered the above error. The same error occurred when I removed the environment and reinstall every packages.

**Expected behavior**
I should see the application launched.

**Screenshots**
Errror code already given above.

**Version (please complete the following information):**
 - Desktop or hosted?: Desktop
 - Browser (if hosted) [e.g. chrome, safari]: N/A
 - Version [e.g. 0.13.0]: 2.3.0

**Additional context**
I'm using conda in Windows 10.

**Comment by atarashansky** [comment]
Hello! Thanks for reporting this.

the tbb layer is a pain in the butt to install so I have a version I’m currently working on that removes this requirement. It should be released in V2.3.2 this weekend.

I’ll keep you posted, and sorry about the difficulties installing.

**Comment by atarashansky** [comment]
Please upgrade excellxgene to 2.3.2:
`pip install excellxgene==2.3.2`

The problem should be resolved. Feel free to reopen this issue if you still have trouble.
